TRANSLATING.md

```diff
@@ -66,3 +66,6 @@ Even though it's required that you read this document in order to contribute wit
 In order to keep consistency in translations, every language should have a translation glossary where common technical terms and terms related to Nextflow have an official translation to be followed by future translators. Ideally, these links should point to an online spreadsheet where anyone can comment and make suggestions, but not edit.
 
 -   [Portuguese](https://docs.google.com/spreadsheets/d/1HUa3BO2kwukhX4EXQ-1blXeP5iueUdM23OwDRpfarDg/edit?usp=sharing)
```

Comment:
shouldn't these glossaries be included in the repo?
why not editable?

Reply:
This early in the translation effort, I think Google Docs provides a better environment for discussion. When it's mature enough, I will move it to Markdown in the repo.